### PR TITLE
Install ShellCheck for the lint job

### PR DESCRIPTION
**Pull request by qbarrand**

### CODE REVIEW

**Comment by ybettan** [comment]
Seems like we have a duplicated effort here https://github.com/openshift/release/pull/24267 :)

File: ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml [hunk 1]
```diff
@@ -16,6 +16,12 @@ images:
 - dockerfile_path: Dockerfile
   from: base
   to: special-resource-operator
+- dockerfile_literal: |-
```
Comment (pacevedom) [inline ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml:19]: Maybe we should add an image coming from src with patch installed? This one could be used for verify, unit testing and linting.
`golangci-lint` makefile target downloads the tool if needed and we would reduce the possible version drift.

Comment (qbarrand) [reply ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml:19]: `src` already has `patch` installed.

**Review by pacevedom — APPROVED** [review]

**Pull request merged**